plugins/modules/web_infrastructure/jira.py

```diff
@@ -223,6 +243,18 @@
     operation: comment
     comment: A comment added by Ansible
 
+- name: Comment on issue with restricted visibility
+  community.general.jira:
+    uri: '{{ server }}'
+    username: '{{ user }}'
+    password: '{{ pass }}'
+    issue: '{{ issue.meta.key }}'
+    operation: comment
+    comment: A comment added by Ansible
+    comment_visibility:
+      type: role
+      value: Developers
+
```

Comment:
That would be nice, but less close to the API. And if jira ever extends the `visibility` structure, this could lead to problems. So maybe keep it as-is :)

Reply:
Yes, this was my thinking as well. If Atlassian introduces additional type, with current proposal the module would not need to be updated.

Reply:
fair enough!